UnityEngine.RuntimePlatform.OSXDashboardPlayer is obsolete

While importing into Unity 2017.2.0f3 I noticed the following in the console:

Assets/3rd Party/SimpleFileBrowser/Scripts/GracesGames/FileBrowser.cs(327,52): error CS0619: `UnityEngine.RuntimePlatform.OSXDashboardPlayer' is obsolete: `Dashboard widget on Mac OS X export is no longer supported in Unity 5.4+

--- a/Assets/SimpleFileBrowser/Scripts/GracesGames/FileBrowser.cs
+++ b/Assets/SimpleFileBrowser/Scripts/GracesGames/FileBrowser.cs
@@ -323,8 +323,7 @@ namespace GracesGames {
 		// Returns whether the application is run on a Mac Operating System
 		private bool IsMacOsPlatform() {
 			return (Application.platform == RuntimePlatform.OSXEditor ||
-			        Application.platform == RuntimePlatform.OSXPlayer ||
-			        Application.platform == RuntimePlatform.OSXDashboardPlayer);
+			        Application.platform == RuntimePlatform.OSXPlayer);
 		}
 		
 		// Creates a directory button given a directory

